stack.yaml

```diff
@@ -8,6 +8,9 @@ packages:
 - '.'
 - '../libraries/Cabal/Cabal'
 
+extra-deps:
+- shake-0.16
+
```

Comment:
Shall we switch to a later Stackage snapshot instead? I fear we might forget to remove this bit later when we switch to a new LTS.

Reply:
I'll take a look

Reply:
the latest LTS-9.5 doesn't support this yet -- I think we can open up an issue so we don't forget this when a new version is rolled out